cumulus/pallets/executive/src/lib.rs

```diff
@@ -175,17 +61,10 @@ impl<T: Config> Pallet<T> {
 	}
 }
 
-/// Main entry point for certain runtime actions as e.g. `execute_block`.
+/// Same semantics with `frame_executive::Executive`.
 ///
-/// Generic parameters:
-/// - `System`: Something that implements `frame_system::Config`
-/// - `Block`: The block type of the runtime
-/// - `Context`: The context that is used when checking an extrinsic.
-/// - `UnsignedValidator`: The unsigned transaction validator of the runtime.
-/// - `AllPalletsWithSystem`: Tuple that contains all pallets including frame system pallet. Will be
-///   used to call hooks e.g. `on_initialize`.
-/// - `OnRuntimeUpgrade`: Custom logic that should be called after a runtime upgrade. Modules are
-///   already called by `AllPalletsWithSystem`. It will be called before all modules will be called.
+/// One extra generic parameter:
+/// - `ExecutivePallet`: Something that implements `Config`.
```

Comment:
Why isn't it called `Config` then? :slightly_smiling_face:

Reply:
Well, I think that's the way following the naming of upstream :P. They use `System` to represent the pallet frame-system, given `Executive` can be confusing here, I changed it to `ExecutivePallet`.

Reply:
Well, it is better I guess, but still confusing :slightly_smiling_face:

Reply:
Feel free to suggest a better one :P

Reply:
I already did, why not call it `Config`?

Reply:
Okay,  didn't realize that 🙈 , so perhaps not a better one. Maybe we should change the comment to make it clearer _`ExecutivePallet`: Something that implements `cirrus_pallet_executive::Config`._ ?

Reply:
Maybe `ExecutiveConfig` then?

Reply:
On the same page, let's take `ExecutiveConfig`.